experiments/pure_mcts/run_experiments_from_files.py

```diff
@@ -0,0 +1,30 @@
+import os
+import shlex
+import subprocess
+import sys
+
+from run_given_experiment import run_from_command_line_inputs
+
+
+def get_docker_commands_from_directories(root_directory):
```

Comment:
What's the difference between this and the other PR?

Reply:
this one has new code which lets us automate the pure MCTS runs